series-tvdb.en.yaml

```diff
@@ -435,6 +450,10 @@ entries:
         anilist-id: 111321
       - season: 3
         anilist-id: 111322
+  - title: "The Slime Diaries"
```

Comment:
On **TheTVDB**, this is "**The Slime Diaries: That Time I Got Reincarnated as a Slime**", only for German and France is it just "**The Slime Diaries**"

Reply:
yep again my bad